test/verify/check-apps

```diff
@@ -82,6 +83,10 @@ class TestApps(PackageCase):
         self.createAppStreamPackage("app-1", "1.0", "1")
         self.enableAppStreamRepo()
 
+        b.click(".app-list tr:contains('An application for testing')")
```

Comment:
You need to use some b.wait_present calls here to make sure the table is loaded before clicking.